src/components/RightSidebar/Participants/ParticipantsTab.vue

```diff
@@ -28,7 +28,7 @@
 			:is-searching="isSearching"
 			@input="handleInput"
 			@abort-search="abortSearch" />
-		<Caption v-if="isSearching && canAdd"
+		<AppNavigationCaption v-if="isSearching && canAdd"
```

Comment:
not imported?

Reply:
Not it's AppNavigation and now you use it in the sidebar... this just shouts for errors popping up in the future